lib/inets/doc/src/httpc.xml

```diff
@@ -647,7 +647,20 @@
         <marker id="get_options"></marker>
       </desc>
     </func>
-     
+
+    <func>
+      <name since="OTP @OTP-18118@">ssl_verify_host_options(WildcardHostName) -> list() </name>
+      <fsummary>Returns ssl options for host verification.</fsummary>
+      <type>
+        <v>WildcardHostName = boolean()</v>
+      </type>
+      <desc>
+        <p>Returns ssl options which can be used to verify the host, with or without
+        wildcard matching of the hostname.
```

Comment:
> `wildcard matching of the hostname`

While reading, I'm not sure about purpose of this wildcard. I'm guessing its a pattern matching with something containing hostname ... but maybe could be explained little more so guessing need could be reduced.
Maybe reference to `public_key:pkix_verify_hostname_match_fun(https)` manual would be good enough?